.github/workflows/CI.yml

```diff
@@ -154,36 +154,30 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python: [
-          { version: '3.7', abi: 'cp37-cp37m' },
-          { version: '3.8', abi: 'cp38-cp38' },
-          { version: '3.9', abi: 'cp39-cp39' },
-          { version: '3.10', abi: 'cp310-cp310' },
-        ]
-        target: [aarch64, armv7, s390x, ppc64le, ppc64]
```

Comment:
Removed `ppc64` since it's a rare architecture and it's unsupported since manylinux_2_24.